microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,225 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 08/18/2023
+ms.date: 12/13/2023
 ---
 # Microsoft Edge Add-ons store developer policies
 
+To develop an extension to be published through the Microsoft Edge Add-ons store, follow these Microsoft Edge Add-ons store developer policies (_Add-ons store developer policies_), which apply to submitting extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and providing extensions through the Microsoft Edge Add-ons store.
+
+The _Microsoft Edge Add-ons store_ is also called the _Microsoft Edge Add-ons website_.
+
 
 <!-- ====================================================================== -->
-## Introduction and Objective of this document
+## Principles
+
+A few principles to get you started:
+
+* Offer unique and distinct value within your extensions for Microsoft Edge.  Provide a compelling reason to download your extensions from the Microsoft Edge Add-ons store (Microsoft Edge Add-ons).
+
+* Do not mislead users about what your extension does, who is offering it, and so on.
 
-Thank you for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
+* Do not attempt to cheat users, the system or the ecosystem.  There is no place in Microsoft Edge Add-ons for any kind of fraud; be it ratings and review manipulation, credit card fraud or other fraudulent activity.
 
-The _Microsoft Edge Add-ons store_ is also called the _Microsoft Edge Add-ons website_.  The _Microsoft Edge Add-ons store developer policies_ are also called the _Microsoft Edge Add-ons website developer policies_.
+Adhering to the Microsoft Edge Add-ons store developer policies helps you make choices that enhance the appeal and audience of your extension.
+
+Your extensions are crucial to the experience of hundreds of millions of users.  We look forward to experiencing what you create and are thrilled to help deliver your extensions to the world.<!-- todo: delete paragraph?  not a principle, not essential, not substantive - fits better in Intro section.  focuses on "we in relation to you", not on technology or policies -->
```

Comment:
```suggestion
```

Reply:
if the article has a statement like this, it would go in Intro section, more than Principles section

Reply:
removed the paragraph